pkg/util/transformer/transformers.go

```diff
@@ -45,5 +45,9 @@ func SetupTransformers(informerFactory informers.SharedInformerFactory, koordInf
 		if err := informer.Informer().SetTransform(transformFn); err != nil {
 			klog.Fatalf("Failed to SetTransform in informer, resource: %v, err: %v", resource, err)
 		}
+		// clean up partial metadata
+		if err := informer.Informer().SetTransform(PartialMetadataRemoveTransform); err != nil {
```

Comment:
If set PartialMetadataRemoveTransform as transformer here, it will replace the exist transformer. It's not the expected.
If you want to do this, you should wrap the transformFn with PartialMetadataRemoveTransform.